### PR TITLE
Explorer Toolbar maximised and minimised 

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Fixes: https://github.com/SigNoz/signoz/issues/4647

https://github.com/SigNoz/signoz/assets/26198062/c356dc9f-69fa-40c4-a426-89974149dae4

### CODE REVIEW

**Comment by GeekBoySupreme** [comment]
@Rajat-Dabade the area to which you can drag the toolbar to collapse it need not be highlighted here. I added that in design just as a reference for what the drag area would look like. 
But this looks super cool. Do we have keyboard shortcuts to toggle these states?

**Comment by Rajat-Dabade** [comment]
@GeekBoySupreme, We will be planning for the keyboard shortcuts for this. I have highlighted the droppable area.

**Review by Vikrant2520 — DISMISSED** [review]

**Review by Vikrant2520 — APPROVED** [review]

File: frontend/src/container/ExplorerOptions/utils.ts [hunk 1]
```diff
@@ -67,3 +68,54 @@ export const generateRGBAFromHex = (hex: string, opacity: number): string =>
...
+				[DataSource.TRACES]: true,
+				[DataSource.LOGS]: true,
+			};
+			localStorage.setItem(
+				LOCALSTORAGE.SHOW_EXPLORER_TOOLBAR,
+				JSON.stringify(parsedShowExplorerToolbar),
+			);
+			return true;
+		}
+		const parsedShowExplorerToolbar = JSON.parse(showExplorerToolbar || '{}');
```
Comment (Vikrant2520) [inline frontend/src/container/ExplorerOptions/utils.ts:93]: wrap this in try catch

File: frontend/src/container/ExplorerOptions/utils.ts [hunk 2]
```diff
@@ -67,3 +68,54 @@ export const generateRGBAFromHex = (hex: string, opacity: number): string =>
...
+			const parsedShowExplorerToolbar: {
+				[DataSource.LOGS]: boolean;
+				[DataSource.TRACES]: boolean;
+				[DataSource.METRICS]: boolean;
+			} = {
+				[DataSource.METRICS]: true,
+				[DataSource.TRACES]: true,
+				[DataSource.LOGS]: true,
+			};
+			localStorage.setItem(
```
Comment (Vikrant2520) [inline frontend/src/container/ExplorerOptions/utils.ts:87]: same here

**Review by YounixM — APPROVED** [review]

File: frontend/src/container/ExplorerOptions/utils.ts (hunk 1 above)
Comment (Rajat-Dabade) [reply frontend/src/container/ExplorerOptions/utils.ts:93]: It's already wrapped inside try catch.

File: frontend/src/container/ExplorerOptions/utils.ts (hunk 2 above)
Comment (Rajat-Dabade) [reply frontend/src/container/ExplorerOptions/utils.ts:87]: Already wrapped.

**Review by Vikrant2520 — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**